utils/network_test.go

```diff
@@ -14,6 +14,8 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+const networkLabel = "network "
```

Comment:
What are you trying to do here?

Reply:
The lint was giving an error about the "network" being repeated several times, and required to create const for them so i did and it passed, is it  a problem?

Reply:
Removing networkLabel and adding //nolint:goconst to line 92 should fix this linting error

Reply:
I actually did it  before adding them, but they kept failing. The lint kept giving an error